RubiksCube/Cube.cs

```diff
@@ -65,13 +65,13 @@ public Cube RotateFront(Rotation direction)
                                      Front.BottomMiddle, Front.MiddleMiddle, Front.TopMiddle,
                                      Front.BottomRight, Front.MiddleRight, Front.TopRight);
 
-                var left = Left.WithRightColumn(Bottom.TopLeft, Bottom.TopMiddle, Bottom.TopRight);
+                var left = Left.WithRightColumn(Bottom.GetTopRow().Rotate(Rotation.Clockwise));
 
-                var right = Right.WithLeftColumn(Top.BottomLeft, Top.BottomMiddle, Top.BottomRight);
+                var right = Right.WithLeftColumn(Top.GetBottomRow().Rotate(Rotation.Clockwise));
 
-                var top = Top.WithBottomRow(Left.BottomRight, Left.MiddleRight, Left.TopRight);
+                var top = Top.WithBottomRow(Left.GetRightColumn().Rotate(Rotation.Clockwise));
```

Comment:
Is this column upside down? Just comparing to bottom which the original code pulls in the opposite order

Reply:
Oddly I thought that as when I first changed around the code the tests failed... so then I convinced myself that I was rotating it wrong. 🤔